src/marks/geo.d.ts

```diff
@@ -8,7 +8,7 @@ export interface GeoOptions extends MarkOptions {
    * A required channel for the geometry to render; defaults to identity,
    * assuming *data* is a GeoJSON object or an iterable of GeoJSON objects.
    */
-  geometry?: ChannelValue;
+  geometry?: ChannelValueSpec | {value: ChannelValue; scale?: null | boolean | "auto" | "projection"};
```

Comment:
I think we should add _projection_ to ScaleName so we can use ChannelValueSpec here. Otherwise you’ll have to also add the label property that was just added. And it’s still the case that this produces a Channel, and hence we need to allow _projection_ in Channel["scale"].